packages/koa-nunjucks/index.js

```diff
@@ -4,19 +4,21 @@ module.exports = (path, globals = {}) => {
   const env = nunjucks.configure([path]);
   Object.keys(globals).forEach(key => env.addGlobal(key, globals[key]));
 
+  const nunjucksRender = (relPath, locals = {}) =>
+    new Promise((resolve, reject) => {
+      env.render(`${relPath}.njk`, locals, (err, html) => {
+        if (err) return reject(err);
+        resolve(html);
+      });
+    });
+
   return async (ctx, next) => {
     if (ctx.render) return next();
-    ctx.render = (relPath, locals = {}) => {
-      const promise = new Promise((resolve, reject) => {
-        env.render(`${relPath}.njk`, locals, (err, html) => {
-          if (err) return reject(err);
-          ctx.set('Content-Type', 'text/html; charset=utf-8');
-          ctx.body = html;
-          resolve();
-        });
-      });
-      return promise;
+    ctx.render = async (relPath, locals = {}) => {
+      ctx.set('Content-Type', 'text/html; charset=utf-8');
+      ctx.body = await nunjucksRender(relPath, locals);
     };
+    ctx.nunjucksRender = nunjucksRender;
```

Comment:
maybe another name for this function, render should be a verb here it is a noun.